### PR TITLE
Add ‘interfaceName’ optional configuration

### PR DESCRIPTION
**Pull request by rotx**
Useful for hosts with multiple network interfaces. See issue #317

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ ffmpegPlatform.prototype.configureAccessory = function(accessory) {
 ffmpegPlatform.prototype.didFinishLaunching = function() {
   var self = this;
   var videoProcessor = self.config.videoProcessor || 'ffmpeg';
+  var interfaceName = self.config.interfaceName;
```
Comment (KhaosT) [inline index.js]: Can you explicitly set a fallback value here since this is optional?

Comment (rotx) [reply index.js]: Yes, any value that evaluates to false should work, e.g. an empty string.

```suggestion
  var interfaceName = self.config.interfaceName || '';
```

Comment (KhaosT) [reply index.js]: Looks good. Can you apply the change to this PR and update the PR? Thanks 🙏

**Comment by KhaosT** [comment]
Thank you :)

**Pull request merged**